HandheldCompanion/Managers/ProcessManager.cs

```diff
@@ -419,6 +419,8 @@ private static ProcessFilter GetFilter(string exec, string path, string MainWind
             case "asmultidisplaycontrol.exe":
             case "lockapp.exe":
             case "asusosd.exe":
+            case "steamwebhelper.exe":
+            case "dwm.exe":

```

Comment:
I just saw this is already under ProcessFilter.Restricted but I can't tell you what the correct placement is